content/metadata.yaml

```diff
@@ -981,12 +981,14 @@ author_info:
 #   twitter: null
 #   affiliations:
 #   - Stanford University
-# - name: Robin van der Lee
-#   orcid: null
-#   github: null
-#   twitter: null
-#   affiliations:
-#   - The University of British Columbia
+- name: Robin van der Lee
+   orcid: 0000-0001-7391-9438
```

Comment:
Can you remove one space before each of the lines from orcid: through the end of the affiliations?